### PR TITLE
Adding Add Properties Module

### PR DESCRIPTION
**Pull request by ghost**
Adding 1 controller: AddProperties.php
Editing 1 exiting model: Property.php
Adding 2 new models (Code Igniter Models not custom): PropertyMedia.php, PropertyDetails.php
To avoid confusion, non-custom models don't have 'Model' in their class name.

### CODE REVIEW

File: app/Controllers/AddProperty.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                  "landSize": "97",
+                  "bedrooms": "5",
+                  "bathrooms": "5",
+                  "elevator": [
+                    "on"
+                  ],
+                  "parking": [
+                    "on"
+                  ]
+                }';
```
Comment (eddiemg10) [inline app/Controllers/AddProperty.php:36]: This would be 1 or 0 instead of on/off

File: app/Models/Property.php [hunk 2]
```diff
@@ -7,7 +7,6 @@ class Property extends Model{
     protected $table = "tbl_property";
     protected $primaryKey = "propertyID";
     /*Spell check these.. Left out some tables as well, confirm with db */
-    protected $allowedFields = ["tenantID","propertyCounty","propertyPhysicalAddress","propertyType", "propertyPrice", "isDeleted"];
-    
-
-}
\ No newline at end of file
+    //This nigga really wrote prOpeRtYpRice and asked us to spell check :')
```
Comment (eddiemg10) [inline app/Models/Property.php:10]: I will not stand for such BS. ratio

**Review by eddiemg10 — CHANGES_REQUESTED** [review]
As we await confirmation, can you find out whether this can be done with the db as it is currently?
Also, how will the newly added Routes affect us if I merged rn?

File: app/Controllers/AddProperty.php (hunk 1 above)
Comment (ghost) [reply app/Controllers/AddProperty.php:36]: The routes only affect the views that controller that I've added. The rest remain unchanged. As for property features, I have ignored it for now until we merge backend and frontend as I'm not sure. The 'propertyFeatures' attribute is not being updated as of now(Check the code it's commented out).

Comment (ghost) [reply app/Controllers/AddProperty.php:36]: t̶h̶e̶ ̶v̶i̶e̶w̶s̶

File: app/Models/Property.php (hunk 2 above)
Comment (ghost) [reply app/Models/Property.php:10]: C'mon...

**Review by eddiemg10 — APPROVED** [review]
Idk what I'm doing, hopefully this works

**Pull request merged**